detekt-rules-naming/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/naming/PackageNamingSpec.kt

```diff
@@ -9,6 +9,15 @@ class PackageNamingSpec : Spek({
 
     describe("PackageNaming rule") {
 
+        it("should ignore the issue by alias suppression") {
+            assertThat(
+                PackageNaming().compileAndLint("""
+                    @file:Suppress("PackageDirectoryMismatch")
+                    package FOO.BAR
+                """.trimIndent())
```

Comment:
The code is automatically trimmed by the `lint` API IMHO. Hence, there's no need for this extra step.

```suggestion
                """)
```